vm/vmContainers_test.go

```diff
@@ -1578,7 +1578,26 @@ func TestStructs(t *testing.T) {
 				A interface{}
 				B interface{}
 			}{A: int64(1), B: int64(3)}}},
-
+		{Script: `a.A = 1.2; a.B = 1.3;`, Input: map[string]interface{}{"a": &struct {
```

Comment:
Test a.A = 1.2; a.B = 1.3 does not seem to test anything new. It also does not seem to add any coverage.